Description: Declare that sjlj.c requires X/Open 7
 Things from the X/Open 7 standard are used. This has to
 be declared when building with -std=c99.
Author: Tobias Hansen <thansen@debian.org>

--- a/libco/sjlj.c
+++ b/libco/sjlj.c
@@ -12,6 +12,7 @@
  */
 
 #define LIBCO_C
+#define _XOPEN_SOURCE 700
 #include "libco.h"
 
 #include <stdlib.h>
